src/bfx_tools/sambamba.ml

```diff
@@ -0,0 +1,36 @@
+open Biokepi_run_environment
+open Common
+
+module Remove = Workflow_utilities.Remove
+
+
+(* Filter language syntax at
+   https://github.com/lomereiter/sambamba/wiki/%5Bsambamba-view%5D-Filter-expression-syntax *)
+let view ~(run_with : Machine.t) ~bam ~filter output_file_path =
```

Comment:
I would create a new type for filters:

```ocaml
type filter = [
| `Raw of string
]
```

so that later, if useful, we can add a better-typed alternative  (like `` `Expression of Ast.t``).